### PR TITLE
mac detector should not match UUIDs anymore

### PR DESCRIPTION
**Pull request by tjungblu**

### CODE REVIEW

File: pkg/obfuscator/mac_address.go [hunk 1]
```diff
@@ -30,7 +30,9 @@ func (m *macAddressObfuscator) ReportingResult() map[string]string {
 }
 
 func NewMacAddressObfuscator() Obfuscator {
-	regex := regexp.MustCompile(`(?:[0-9a-fA-F]([:-])?){12}`)
+	// this regex differs from the standard `(?:[0-9a-fA-F]([:-])?){12}`, to not match very frequently happening UUIDs in K8s
+	// the main culprit is the support for squashed MACs like '69806FE67C05', which won't be supported with the below
+	regex := regexp.MustCompile(`([0-9a-fA-F]{2}[:-][0-9a-fA-F]{2}[:-][0-9a-fA-F]{2}[:-][0-9a-fA-F]{2}[:-][0-9a-fA-F]{2}[:-][0-9a-fA-F]{2})`)
```
Comment (arjunrn) [inline pkg/obfuscator/mac_address.go:35]: This can probably be simplified as:
`([0-9a-fA-F]{2}[:-]){5}[0-9a-fA-F]{2}`

Comment (tjungblu) [reply pkg/obfuscator/mac_address.go]: yep, good one

**Pull request merged**